packages/protocol/contracts/governance/SlasherUtil.sol

```diff
@@ -21,6 +21,14 @@ contract SlasherUtil is Ownable, Initializable, UsingRegistry, UsingPrecompiles
 
   event SlashingIncentivesSet(uint256 penalty, uint256 reward);
 
+  /**
```

Comment:
I don't think we want this SlasherUtil to be versioned as multiple versioned contracts inherit